testplan/common/entity/base.py

```diff
@@ -207,7 +207,7 @@ def start(self):
                     pass
 
             else:
-                resource.logger.user_info("%s started", resource)
```

Comment:
numb these to info for now, until we have per-entity logging level. Otherwise the console looks too busy with LocalRunner, Environments and other things that are of no concern of users.